scripts/release/create_release_PR_commit.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+set -eu
+
+#Versions
+#Expected to change more often , when we run this script.
+NEW_VERSION="2.15.3"
+CURRENT_VERSION="2.15.2"
+LAST_VERSION="2.15.1"
+RELEASE_BRANCH="2.15.x"
+RELEASE_ISSUE="613"
```

Comment:
Use of these variables:

`CURRENT_VERSION` & `LAST_VERSION` are used to form the names of update & downgrade SQL files we need. And also to update the `version.config` in e.g. `2.15.x` branch
`RELEASE_BRANCH `is used for checking out e.g. the `release-2.15.3` branch from `2.15.x` one, instead of `main`. 
`RELEASE_ISSUE` is being no more used in the script , so I'll remove it.

Maybe we can derive these variables from the context of `main` branch as:

```
CURRENT_VERSION=`tail -1 version.config | cut -d ' ' -f 3`
cd sql/updates
LAST_VERSION=`ls |grep $CURRENT_VERSION.sql |cut -d '-' -f 1`
CURRENT_PATCH_VERSION=`echo $CURRENT_VERSION | cut -d '.' -f 3`
RELEASE_BRANCH=`echo "${CURRENT_VERSION/%.$CURRENT_PATCH_VERSION/.x}"`
```

Reply:
Resolving this, as it's fixed in the script now.